tools/actorfixer.py

```diff
@@ -16,6 +16,8 @@
     "func_8012404c": "Player_RemoveMask",
     "Actor_SpawnWithParentAndCutscene": "Actor_SpawnAsChildAndCutscene",
     "Actor_SpawnWithParent": "Actor_SpawnAsChild",
+    "PLAYER": "GET_PLAYER(globalCtx)",
+    "ACTIVE_CAM": "GET_ACTIVE_CAM(globalCtx)",
```

Comment:
This is a bad idea, because `GET_PLAYER(globalCtx)` contains the word `PLAYER`, so running this script a second time would produce `GET_GET_PLAYER(globalCtx)(globalCtx)` and so on